monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+# **Monero: Un refugio a las severas leyes de decomiso civil (parte 1)** 
+*04/15/19*  
+Obiter Cryptum  
+
+
+El atropello excesivo, sistemático y atroz por parte del gobierno y de terceros sucede todos los días, y mientras que usted personalmente no lo experimenta, esta vivo y floreciendo. Los derechos fundamentales, los que se nos garantizan a nosotros los seres humanos a través de [La Declaración Universal de Derechos Humanos](https://www.un.org/es/universal-declaration-human-rights/) y las constituciones estatales, están siendo reducidos y usurpados, ya sea de manera evidente u oculta. 
+
+La autonomía, la elección y la expresión han sido elegidos por todos menos por las [3 billones de personas o más](https://www.bancomundial.org/es/news/press-release/2018/10/17/nearly-half-the-world-lives-on-less-than-550-a-day) viviendo con menos de $2.50 USD por día; sea que hayan sido por extralimitaciones estatales, tropiezos o por censura. 
+
+*¿Pero, y si eso no debió de haber sido el caso?* 

```

Comment:
A little suggestion, the sentence is not wrong:

+ _**Pero, ¿y si ese no fuese el caso?**_

Reply:
better, changed*